modules/microshift-ops-config-embed-ostree.adoc

```diff
@@ -0,0 +1,176 @@
+//Q: Do we use this procedure for homegrown operators if we have a local, file-based catalog that we made?
+
+.Prerequisites
+* You have a {op-system-ostree} image.
+* Networking is configured for offline use.
+* You completed the oc mirror dry run procedure.
+
+.Procedure
+
+. Create a `CatalogSource` custom resource by using the following example YAML:
+//add cat command here to make the file
++
+.Example `CatalogSource` YAML
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1alpha1
+kind: CatalogSource
+metadata:
+  name: cs-redhat-operator-index
+  namespace: openshift-marketplace <1>
+spec:
+  image: registry.example.com/redhat/redhat-operator-index:v4.14
+  sourceType: grpc
+  displayName:
+  publisher:
+  grpcPodConfig:
+    securityContextConfig: baseline <2>
+  updateStrategy:
+    registryPoll:
+      interval: 60m
+----
+<1> The global namespace. Setting the `metadata.namespace` to `openshift-marketplace` enables the catalog to run in all namespaces. Subscriptions in any namespace can reference catalogs created in the `openshift-marketplace` namespace.
+<2> The default value of `securityContextConfig`.
+
+. Change the `openshift-marketplace` namespace label `pod-security.kubernetes.io/enforce` from `restricted` to `privileged` in the Catalog Source custom resource (CR):
++
+.Example namespace configuration
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1alpha1
+kind: CatalogSource
+metadata:
+  name: cs-redhat-operator-index
+  namespace: openshift-marketplace
+spec:
+  image: registry.example.com/redhat/redhat-operator-index:v4.14
+  sourceType: grpc
+  displayName:
+  publisher:
+  grpcPodConfig:
+    securityContextConfig: privileged <1>
+  updateStrategy:
+    registryPoll:
+      interval: 60m
+----
+<1> Update default `baseline` value to `privaleged`.
+
+. Get the SHA of the catalog index image by running the following command:
++
+[source,terminal]
+----
+$ skopeo inspect docker://registry.example.com/redhat/redhat-operator-index:v4.14 | jq `.Digest` <1>
+----
+<1> Pass the value from the `spec.image` field in the Catalog Source CR.
++
+.Example output
+[source,terminal]
+----
+"sha256:7a76c0880a839035eb6e896d54ebd63668bb37b82040692141ba39ab4c539bc6"
+----
+
+[IMPORTANT]
+====
+You must use the SHA instead of a tag in your catalog CR or the pod fails to start.
+====
+
+. Add the SHA of the catalog index commit to the Catalog Source (CR) by editing the YAML:
++
+.Example namespace `spec.image` configuration
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1alpha1
+kind: CatalogSource
+metadata:
+  name: cs-redhat-operator-index
+  namespace: openshift-marketplace
+spec:
+  image: registry.example.com/redhat/redhat-operator-index@sha256:7a76c0880a839035eb6e896d54ebd63668bb37b82040692141ba39ab4c539bc6 <1>
+  sourceType: grpc
+  displayName:
+  publisher:
+  grpcPodConfig:
+    securityContextConfig: privileged
```

Comment:
@ShaunaDiaz no privileged supported, so suggest to follow the last two step output: no it or set it as restricted